homeassistant/components/config/entity_registry.py

```diff
@@ -114,9 +115,18 @@ async def websocket_update_entity(hass, connection, msg):
             websocket_api.error_message(msg["id"], "invalid_info", str(err))
         )
     else:
-        connection.send_message(
-            websocket_api.result_message(msg["id"], _entry_ext_dict(entry))
+        result = websocket_api.result_message(
```

Comment:
Don't create a result and then mutate the result. It means you're assuming that the result message will stay the same. 

Instead, build up your response, then wrap it. Use `connection.send_result(msg['id'], result)` also, don't use `result_message` or `error_message` anymore.

Reply:
Fixed to build up the response, wrap it, then use `connection.send_result(msg['id'], result)` to send it off.

Not using `result_message` or `error_message` would be better as a separate PR.